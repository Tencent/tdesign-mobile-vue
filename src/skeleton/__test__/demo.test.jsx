/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import animationVue from '@/skeleton/demos/animation.vue';
import cellGroupVue from '@/skeleton/demos/cell-group.vue';
import gridVue from '@/skeleton/demos/grid.vue';
import imageGroupVue from '@/skeleton/demos/image-group.vue';
import mobileVue from '@/skeleton/demos/mobile.vue';
import themeVue from '@/skeleton/demos/theme.vue';

const mapper = {
  animationVue,
  cellGroupVue,
  gridVue,
  imageGroupVue,
  mobileVue,
  themeVue,
};

describe('Skeleton', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Skeleton ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
