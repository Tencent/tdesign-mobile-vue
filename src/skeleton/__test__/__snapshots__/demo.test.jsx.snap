// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Skeleton > Skeleton animationVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    data-v-5431dabc=""
  >
    <div
      class="demo-section__desc"
      data-v-5431dabc=""
    >
      渐变加载效果
    </div>
    <div
      class="demo-section__content"
      data-v-5431dabc=""
    >
      <div
        class="t-skeleton"
        data-v-5431dabc=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
            style="width: 55%;"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
    </div>
  </div>
  <div
    data-v-5431dabc=""
  >
    <div
      class="demo-section__desc"
      data-v-5431dabc=""
    >
      闪烁加载效果
    </div>
    <div
      class="demo-section__content"
      data-v-5431dabc=""
    >
      <div
        class="t-skeleton"
        data-v-5431dabc=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
            style="width: 55%;"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
    </div>
  </div>
  
</div>
`;

exports[`Skeleton > Skeleton cellGroupVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="group"
    data-v-b5595f68=""
  >
    <div
      class="t-skeleton group-avatar"
      data-v-b5595f68=""
    >
      
      
      <div
        class="t-skeleton__row"
      >
        
        <div
          class="t-skeleton__col t-skeleton--type-circle t-skeleton--animation-none"
          style="width: 48px; height: 48px;"
        >
          <!---->
        </div>
        
      </div>
      
      
    </div>
    <div
      class="t-skeleton group-content"
      data-v-b5595f68=""
    >
      
      
      <div
        class="t-skeleton__row"
      >
        
        <div
          class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          style="width: 50%;"
        >
          <!---->
        </div>
        
      </div>
      <div
        class="t-skeleton__row"
      >
        
        <div
          class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          style="width: 100%;"
        >
          <!---->
        </div>
        
      </div>
      
      
    </div>
  </div>
  <div
    class="group"
    data-v-b5595f68=""
  >
    <div
      class="t-skeleton group-avatar"
      data-v-b5595f68=""
    >
      
      
      <div
        class="t-skeleton__row"
      >
        
        <div
          class="t-skeleton__col t-skeleton--type-rect t-skeleton--animation-none"
          style="width: 48px; height: 48px;"
        >
          <!---->
        </div>
        
      </div>
      
      
    </div>
    <div
      class="t-skeleton group-content"
      data-v-b5595f68=""
    >
      
      
      <div
        class="t-skeleton__row"
      >
        
        <div
          class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          style="width: 50%;"
        >
          <!---->
        </div>
        
      </div>
      <div
        class="t-skeleton__row"
      >
        
        <div
          class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          style="width: 100%;"
        >
          <!---->
        </div>
        
      </div>
      
      
    </div>
  </div>
  
</div>
`;

exports[`Skeleton > Skeleton gridVue demo works fine 1`] = `
<div
  class="t-skeleton"
>
  
  
  <div
    class="t-skeleton__row"
  >
    
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 48px; border-radius: 6px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 48px; border-radius: 6px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 48px; border-radius: 6px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 48px; border-radius: 6px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 48px; border-radius: 6px;"
    >
      <!---->
    </div>
    
  </div>
  <div
    class="t-skeleton__row"
  >
    
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 16px; border-radius: 3px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 16px; border-radius: 3px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 16px; border-radius: 3px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 16px; border-radius: 3px;"
    >
      <!---->
    </div>
    <div
      class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      style="width: 48px; height: 16px; border-radius: 3px;"
    >
      <!---->
    </div>
    
  </div>
  
  
</div>
`;

exports[`Skeleton > Skeleton imageGroupVue demo works fine 1`] = `
<div
  class="group"
  data-v-ac6b2376=""
>
  <div
    class="t-skeleton"
    data-v-ac6b2376=""
  >
    
    
    <div
      class="t-skeleton__row"
    >
      
      <div
        class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
        style="width: 163.5px; height: 163.5px; border-radius: 12px;"
      >
        <!---->
      </div>
      
    </div>
    <div
      class="t-skeleton__row"
    >
      
      <div
        class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      >
        <!---->
      </div>
      
    </div>
    <div
      class="t-skeleton__row"
    >
      
      <div
        class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
        style="width: 61%;"
      >
        <!---->
      </div>
      
    </div>
    
    
  </div>
  <div
    class="t-skeleton"
    data-v-ac6b2376=""
  >
    
    
    <div
      class="t-skeleton__row"
    >
      
      <div
        class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
        style="width: 163.5px; height: 163.5px; border-radius: 12px;"
      >
        <!---->
      </div>
      
    </div>
    <div
      class="t-skeleton__row"
    >
      
      <div
        class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
      >
        <!---->
      </div>
      
    </div>
    <div
      class="t-skeleton__row"
    >
      
      <div
        class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
        style="width: 61%;"
      >
        <!---->
      </div>
      
    </div>
    
    
  </div>
</div>
`;

exports[`Skeleton > Skeleton mobileVue demo works fine 1`] = `
<div
  class="tdesign-mobile-demo"
  data-v-9b6b90cc=""
>
  <h1
    class="title"
    data-v-9b6b90cc=""
  >
    Skeleton 骨架屏
  </h1>
  <p
    class="summary"
    data-v-9b6b90cc=""
  >
    当网络较慢时，在页面真实数据加载之前，给用户展示出页面的大致结构。
  </p>
  <div
    class="tdesign-mobile-demo-block"
    data-v-9b6b90cc=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        01 类型
      </h2>
      <!--v-if-->
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      
      <div
        data-v-7e69da81=""
      >
        <div
          class="demo-section__desc"
          data-v-7e69da81=""
        >
          头像骨架屏
        </div>
        <div
          class="demo-section__content"
          data-v-7e69da81=""
        >
          <div
            class="t-skeleton"
            data-v-7e69da81=""
          >
            
            
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-circle t-skeleton--animation-none"
                style="width: 48px; height: 48px;"
              >
                <!---->
              </div>
              
            </div>
            
            
          </div>
        </div>
      </div>
      <div
        data-v-7e69da81=""
      >
        <div
          class="demo-section__desc"
          data-v-7e69da81=""
        >
          图片骨架屏
        </div>
        <div
          class="demo-section__content"
          data-v-7e69da81=""
        >
          <div
            class="t-skeleton"
            data-v-7e69da81=""
          >
            
            
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-rect t-skeleton--animation-none"
                style="width: 72px; height: 72px;"
              >
                <!---->
              </div>
              
            </div>
            
            
          </div>
        </div>
      </div>
      <div
        data-v-7e69da81=""
      >
        <div
          class="demo-section__desc"
          data-v-7e69da81=""
        >
          文本骨架屏
        </div>
        <div
          class="demo-section__content"
          data-v-7e69da81=""
        >
          <div
            class="t-skeleton"
            data-v-7e69da81=""
          >
            
            
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
                style="width: 24%; height: 16px; margin-right: 16px;"
              >
                <!---->
              </div>
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
                style="width: 76%; height: 16px;"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              >
                <!---->
              </div>
              
            </div>
            
            
          </div>
        </div>
      </div>
      <div
        data-v-7e69da81=""
      >
        <div
          class="demo-section__desc"
          data-v-7e69da81=""
        >
          段落骨架屏
        </div>
        <div
          class="demo-section__content"
          data-v-7e69da81=""
        >
          <div
            class="t-skeleton"
            data-v-7e69da81=""
          >
            
            
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
                style="width: 55%;"
              >
                <!---->
              </div>
              
            </div>
            
            
          </div>
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-9b6b90cc=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        单元格骨架屏
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      
      <div
        class="group"
        data-v-b5595f68=""
      >
        <div
          class="t-skeleton group-avatar"
          data-v-b5595f68=""
        >
          
          
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-circle t-skeleton--animation-none"
              style="width: 48px; height: 48px;"
            >
              <!---->
            </div>
            
          </div>
          
          
        </div>
        <div
          class="t-skeleton group-content"
          data-v-b5595f68=""
        >
          
          
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 50%;"
            >
              <!---->
            </div>
            
          </div>
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 100%;"
            >
              <!---->
            </div>
            
          </div>
          
          
        </div>
      </div>
      <div
        class="group"
        data-v-b5595f68=""
      >
        <div
          class="t-skeleton group-avatar"
          data-v-b5595f68=""
        >
          
          
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-rect t-skeleton--animation-none"
              style="width: 48px; height: 48px;"
            >
              <!---->
            </div>
            
          </div>
          
          
        </div>
        <div
          class="t-skeleton group-content"
          data-v-b5595f68=""
        >
          
          
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 50%;"
            >
              <!---->
            </div>
            
          </div>
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 100%;"
            >
              <!---->
            </div>
            
          </div>
          
          
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-9b6b90cc=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        宫格骨架屏
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      <div
        class="t-skeleton"
        data-v-9b6b90cc=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 48px; border-radius: 6px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 48px; border-radius: 6px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 48px; border-radius: 6px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 48px; border-radius: 6px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 48px; border-radius: 6px;"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 16px; border-radius: 3px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 16px; border-radius: 3px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 16px; border-radius: 3px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 16px; border-radius: 3px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 48px; height: 16px; border-radius: 3px;"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-9b6b90cc=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        图文组合骨架屏
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      <div
        class="group"
        data-v-9b6b90cc=""
        data-v-ac6b2376=""
      >
        <div
          class="t-skeleton"
          data-v-ac6b2376=""
        >
          
          
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 163.5px; height: 163.5px; border-radius: 12px;"
            >
              <!---->
            </div>
            
          </div>
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            >
              <!---->
            </div>
            
          </div>
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 61%;"
            >
              <!---->
            </div>
            
          </div>
          
          
        </div>
        <div
          class="t-skeleton"
          data-v-ac6b2376=""
        >
          
          
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 163.5px; height: 163.5px; border-radius: 12px;"
            >
              <!---->
            </div>
            
          </div>
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            >
              <!---->
            </div>
            
          </div>
          <div
            class="t-skeleton__row"
          >
            
            <div
              class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
              style="width: 61%;"
            >
              <!---->
            </div>
            
          </div>
          
          
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
    data-v-9b6b90cc=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        02 组件动效
      </h2>
      <!--v-if-->
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      
      <div
        data-v-5431dabc=""
      >
        <div
          class="demo-section__desc"
          data-v-5431dabc=""
        >
          渐变加载效果
        </div>
        <div
          class="demo-section__content"
          data-v-5431dabc=""
        >
          <div
            class="t-skeleton"
            data-v-5431dabc=""
          >
            
            
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-gradient"
                style="width: 55%;"
              >
                <!---->
              </div>
              
            </div>
            
            
          </div>
        </div>
      </div>
      <div
        data-v-5431dabc=""
      >
        <div
          class="demo-section__desc"
          data-v-5431dabc=""
        >
          闪烁加载效果
        </div>
        <div
          class="demo-section__content"
          data-v-5431dabc=""
        >
          <div
            class="t-skeleton"
            data-v-5431dabc=""
          >
            
            
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
              >
                <!---->
              </div>
              
            </div>
            <div
              class="t-skeleton__row"
            >
              
              <div
                class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-flashed"
                style="width: 55%;"
              >
                <!---->
              </div>
              
            </div>
            
            
          </div>
        </div>
      </div>
      
      
    </div>
  </div>
</div>
`;

exports[`Skeleton > Skeleton themeVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    data-v-7e69da81=""
  >
    <div
      class="demo-section__desc"
      data-v-7e69da81=""
    >
      头像骨架屏
    </div>
    <div
      class="demo-section__content"
      data-v-7e69da81=""
    >
      <div
        class="t-skeleton"
        data-v-7e69da81=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-circle t-skeleton--animation-none"
            style="width: 48px; height: 48px;"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
    </div>
  </div>
  <div
    data-v-7e69da81=""
  >
    <div
      class="demo-section__desc"
      data-v-7e69da81=""
    >
      图片骨架屏
    </div>
    <div
      class="demo-section__content"
      data-v-7e69da81=""
    >
      <div
        class="t-skeleton"
        data-v-7e69da81=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-rect t-skeleton--animation-none"
            style="width: 72px; height: 72px;"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
    </div>
  </div>
  <div
    data-v-7e69da81=""
  >
    <div
      class="demo-section__desc"
      data-v-7e69da81=""
    >
      文本骨架屏
    </div>
    <div
      class="demo-section__content"
      data-v-7e69da81=""
    >
      <div
        class="t-skeleton"
        data-v-7e69da81=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 24%; height: 16px; margin-right: 16px;"
          >
            <!---->
          </div>
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 76%; height: 16px;"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
    </div>
  </div>
  <div
    data-v-7e69da81=""
  >
    <div
      class="demo-section__desc"
      data-v-7e69da81=""
    >
      段落骨架屏
    </div>
    <div
      class="demo-section__content"
      data-v-7e69da81=""
    >
      <div
        class="t-skeleton"
        data-v-7e69da81=""
      >
        
        
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
          >
            <!---->
          </div>
          
        </div>
        <div
          class="t-skeleton__row"
        >
          
          <div
            class="t-skeleton__col t-skeleton--type-text t-skeleton--animation-none"
            style="width: 55%;"
          >
            <!---->
          </div>
          
        </div>
        
        
      </div>
    </div>
  </div>
  
</div>
`;
